### PR TITLE
Update defaults for batch_size and num_workers

### PR DESCRIPTION
**Pull request by abhibyreddi**
Update command in deployment file with batch_size and num_workers set to values that gave us best egress BW for 150MB dataset

Also, `Unet3DLightning.training_step` now returns a scalar (that represents loss) instead of None to skip actual training. This is because pytorch lightning does not allow skipping training when run on multiple nodes (returning None on single node training works just fine!)

- [X] Tests pass; deployed on a 10 node cluster and verified that the workload finished successfully
- [X] Appropriate changes to documentation are included in the PR - N/A

### CODE REVIEW

File: demo/lightning/image-segmentation/model.py [hunk 1]
```diff
@@ -56,7 +56,8 @@ def configure_optimizers(self):
     def training_step(self, train_batch, batch_idx):
         images, labels = train_batch
         if self.benchmark:
-            return None
+            # Return a scalar of value 1 to speed up training_step.
```
Comment (MattIrv) [inline demo/lightning/image-segmentation/model.py:59]: nit: This comment doesn't really explain why this speeds it up compared to returning None. Is there any way to add that, either to the comment or to the PR description?

**Review by MattIrv — APPROVED** [review]

File: demo/lightning/image-segmentation/model.py (hunk 1 above)
Comment (abhibyreddi) [reply demo/lightning/image-segmentation/model.py]: Updated the comment just now :)

Comment (abhibyreddi) [reply demo/lightning/image-segmentation/model.py]: Also updated PR description.
